lib/teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(age, name, parent_permission, specialization)
```

Comment:
- [ ] `name` and `parent_permission` should be optional and have default values of "Unknown" and 'true'`
```def initialize(age, specialization, name = "Unknown", parent_permission = true)```